public/content/roadmap/index.md

```diff
@@ -66,7 +66,7 @@ The roadmap is mostly the result of years of work by researchers and developers
 </RoadmapImageContent>
 
 <InfoBanner mb={8}>
-  <h4 style="margin-top: 0">What was ETH2?</h4>
```

Comment:
@pettinarip This was a result of compiler errors I was getting. The build failed because it was expecting an object for the `style` attribute, not a string.

Reply:
Otherwise I agree =) If you know which ts setting we can tweak to fix this, we can use that and revert this.

I could post a PR to the Gatsby repo, but it's not required there, so given it's only needed here and seems to be breaking without it I went with just refactoring them.

@minimalsm On this note... can you recall if this would cause any Crowdin issues? I don't think it should, but can make the syntax a bit more complex looking.

Reply:
Ah okok, thats ok then. If we are fixing an issue that only happens in this repo then its ok. Its part of the migration.